### PR TITLE
Add django 1.8/1.9 compatibility. Drop unsupported django versions support

### PR DESCRIPTION
**Pull request by kmmbvnr**
For #72 #75

It's time to drop old django's

download django django

### CODE REVIEW

**Comment by kmmbvnr** [comment]
Can you please merge  this and release on PyPI? 

please-640x288

**Comment by mauricioabreu** [comment]
LGTM :+1:

**Comment by berkersonmez** [comment]
Can this be merged please?

File: parsley/widgets.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
 class ParsleyChoiceFieldRendererMixin(object):
-    def __iter__(self):
-        for i, choice in enumerate(self.choices):
-            yield self[i]
+    def __init__(self, name, value, attrs, choices):
+        self.name = name
+        self.value = value
+        self.attrs = attrs
+        self._choises = choices
```
Comment (mauricioabreu) [inline parsley/widgets.py]: I think there is a typo here.

**Pull request merged**